lib/router/unrecognized-url-error.js

```diff
@@ -5,9 +5,25 @@ import { oCreate } from './utils';
   handlers for failed transitions.
  */
```

Comment:
This description doesn't make any sense to me, should this be replaced with something else?